internal/gatewayapi/testdata/clienttrafficpolicy-preserve-case.in.yaml

```diff
@@ -0,0 +1,36 @@
+clientTrafficPolicies:
+- apiVersion: gateway.envoyproxy.io/v1alpha1
+  kind: ClientTrafficPolicy
+  metadata:
+    namespace: envoy-gateway
+    name: target-gateway-1
+  spec:
+    http1:
+      enableTrailers: true
```

Comment:
nit: not needed

Reply:
You're right, it's not technically needed. It doesn't hurt though, so I left it in since I noticed that I didn't remove it only after I had already pushed the PR.